### ISSUE TITLE
[CoE Starter Kit - BUG]: Audit Logs Custom Connector issues on clean install

### ISSUE THREAD

**Issue by bartsneary** [microsoft/coe-starter-kit#1014]
### Describe the issue

I reset an environment and installed the core solution (2.75).  All was going well until I got to the custom connector configuration.  When I click to edit the custom connector, depending on how to get to it, I get one of two errors.  I either get:
- Failed to download original Swagger file of the custom connector or
- Could not retrieve custom connector data or
- The request failed with error: '{"error":{"code":"0x80040217","message":"systemuser With Id = dca5a216-2463-eb11-b0b0-000d3a98cbf5 Does Not Exist"}}'. The tracking Id is '13d7850a-3836-46f5-ad2c-9f46cbdc781d' if I try to establish a connection with the connector

Just in case it was something with the install, I reapplied the solution in the environment, but that didn't change anything.

### Expected Behavior

I expected to be able to edit the connector to add in the required details for my tenant so I could create a connection and load the audit logs solution.  I couldn't do any of those things.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

2.75

### What app or flow are you having the issue with?

Office 365 Management API Custom Connector

### Steps To Reproduce

1) Create a new clean environment
2) Install Core 2.75
3) Attempt to configure the custom connector

**Comment by manuelap-msft** [comment]
Hello,

we have seen the "systemuser With Id not found" error before, but it is always an issue with the environment and not with the CoE Starter Kit. I have tested setting up the custom connector in our Test/Prod deployment environments where we try both upgrades and clean install the managed solution and it worked there. I think it may be an issue with your environment?

Thank you
Manuela

**Comment by manuelap-msft** [comment]
I would check the systemusers table of the enviornment and see what users are listed there - it may be that your user has not been added there.

**Comment by bartsneary** [comment]
I just got another weird error when trying to share a Canvas App from this environment saying that user with id = "dca5a216-2463-eb11-b0b0-000d3a98cbf5" does not exist, which is one of the messages I got with the custom connector also.  Given that I reset this environment, is it possible those assets (apps, flows, and custom connectors) somehow persisted through the reset operation and were owned by a user that had a different ID behind the scenes than the same user does now in this environment and permissions are now causing these issues?  Weird stuff!  I can't work on it more until tomorrow now, but may have to make an entirely new environment to get past this...
